### ISSUE TITLE
Zentangle creation error

### ISSUE THREAD

**Issue by schrum2** [sarahkfriday/CPPNArtEvolutionandWFC#3]
It turns out that if you start with a fresh empty zentangle directory, the code will crash whether you are using color or black and white (at least when 5 images are selected). The only reason it worked before was because previously saved images were being used.

I think part of the issue is that the variable fourthImage is not even being used, though I tried a naive fix and it still didn't work. I think this code needs to be cleaned up quite a bit to make #1 work, so hopefully this issue will be solved when you address that issue.

**Comment by krolikoa** [comment]
This issue ended up being with a for loop that only ran for half of the number of selections made. I commented out the loop but didn't remove it in case there was an important reason why it was there.

**Comment by schrum2** [comment]
Great @krolikoa ! This issue should definitely be closed.

However, I think we are missing a feature that used to be present in the code after I edited it. My edits added the ability to have multiple different images merged into the same background template using WFC. I think this was the purpose of the loop ... half of the images were picked to be in one background pattern, and the other half were mixed up in the second pattern. Then there two patterns were used to correspond to the black/white parts of the main image.

We're missing that ability now ... we probably want to add it back eventually. I'll make an issue for this, but for now please focus on #1 and #2 .

**Comment by schrum2** [comment]
Actually, I just went back to the MM-NEAT repo (https://github.com/schrum2/MM-NEAT in dev_zentangle branch) and tried my old version, and the results kind of look like crap:

![zentangle](https://user-images.githubusercontent.com/8397236/62340477-4f076900-b4a5-11e9-88e7-93b50ce3284c.png)

Although it is cool that multiple types of images can be combined into a single background like the following,

![picbreederZentangle1](https://user-images.githubusercontent.com/8397236/62340534-870eac00-b4a5-11e9-9ca6-ce95073e7cbf.jpg)

It just ends up looking too chaotic. I prefer your new approach, where instead of having lots of variation in the backgrounds, we just have more distinct regions in the image, but each still with a single pattern.

Long story short, good job closing this issue! (again)
